mod_reforged/hooks/skills/perks/perk_mastery_spear.nut

```diff
@@ -1,8 +1,118 @@
 ::Reforged.HooksMod.hook("scripts/skills/perks/perk_mastery_spear", function(q) {
-	q.onAfterUpdate <- function( _properties )
+	q.m.IsSpent <- true;
+	q.m.DamageMult <- 0.75;
+	q.m.Skills <- [
```

Comment:
This is a poor name choice as it can be confused with the often used `this.getContainer().m.Skills` snippet.
I suggest changing it into something like `ValidSkills`